src/library-authoring/create-library/data/types.d.ts

```diff
@@ -0,0 +1,5 @@
+export type CreateContentLibraryDto = {
```

Comment:
This can be an `interface` not a `type` (it's better to use interface when you can), and you can just put it in `api.ts` above the function that uses it. 

```ts
export interface CreateContentLibraryArgs {
  title: string;
  org: string;
  slug: string;
}
``

Reply:
Done 8d712972dc53e00ef9d411a1d7b3ff919a53027a!